lib/v2/wmpvp/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'wmpvp.com': {
+        _name: '完美世界电竞',
+        '.': [
+            {
+                title: '资讯',
+                docs: 'https://docs.rsshub.app/routes/game#wmpvp-zi-xun',
```

Comment:
```suggestion
                docs: 'https://docs.rsshub.app/routes/game#wan-mei-shi-jie-dian-jing',
```

Reply:
Route has been renamed from `wmpvp-zi-xun` to `wan-mei-shi-jie-dian-jing-zi-xun`